src/main/java/org/isf/lab/manager/LabManager.java

```diff
@@ -92,7 +92,7 @@ protected void validateLaboratory(Laboratory laboratory) throws OHDataValidation
 		// Check Patient
 		if (GeneralData.LABEXTENDED && laboratory.getPatient() == null) {
 			errors.add(new OHExceptionMessage("patientNullError",
```

Comment:
why in this case we left the `"patientNullError"` while in other cases we replaced with `MessageBundle.getMessage("angal.common.error.title")`?  According to my understanding the "title" would be useful as univocal "error code" in order to identifiy the exception, not like a common title for every error

Reply:
Yes it should be but it wasn't my focus when I was changing to a common "select patient message"; then I do the lab dialog this will be accounted for.

Reply:
I made all the title changes in the CORE project.

Reply:
Thanks!